lib/workers/repository/process/lookup/generate.ts

```diff
@@ -80,6 +80,14 @@ export async function generateUpdate(
   update.updateType =
     update.updateType ??
     getUpdateType(config, versioningApi, currentVersion, newVersion);
+  if (versioningApi.isBreaking) {
+    // This versioning scheme has breaking awareness
+    update.isBreaking = versioningApi.isBreaking(currentVersion, newVersion);
+  } else {
+    // This versioning scheme does not have breaking awareness - assume only major updates are breaking
+    // Updates from or to unstable releases should be treated as breaking too, but we should not add that as default behavior until we stop treating non-LTS
```

Comment:
```suggestion
    // Updates from, or to, unstable releases should be treated as breaking too.
    // But we should not add that as default behavior until we stop treating non-LTS versions ...INSERT EXPLANATION HERE...
```

Let's split this into two sentences, and add some commas to help read the first sentence.

I think the last sentence is missing information about what needs to change to `non-LTS`, before you can "treat updates from, or to unstable releases as breaking". So I've added some placeholder text for you to fill out. :wink: